log10/feedback/feedback.py

```diff
@@ -111,3 +160,40 @@ def _get_feedback_list(offset, limit, task_id):
         return []
 
     return feedback_data
+
+
+def _format_graphql_node(node):
+    return {
+        "id": node["id"],
+        "json_values": node["jsonValues"],
+        "task_id": node["task"]["id"],
+        "task_name": node["task"]["name"],
+        "matched_completion_ids": [c["id"] for c in node["completions"]],
+    }
+
+
+def _get_feedback_list_graphql(page, limit, task_id, filter):
+    feedback_data = []
+    current_page = page if page else 1
+    if limit:
+        limit = int(limit)
```

Comment:
nit: thinking to add the default 50 value here. 
```
else:
    limit = 50
```
so that line 183 could be `fetch_limit = limit` in line 183. or just directly use limit in the while loop